dispatcher.py

```diff
@@ -46,35 +46,6 @@
 from listeners import StatisticsWatcher
 
 
-class TcpPortDispatcher:
```

Comment:
How about removing of `utils.find_port()`?

Reply:
`find_port()` is still used in inspector.py.

Reply:
And there is the same race here, right? See, it is the simplest place: here we acquire a port, listen and use it later. Nothing prevent us from binding to zero, listen, get the real port and add it into the environment. Moreover, nothing actually prevent us from using Unix sockets here. A worker has one inspector, a worker has one working directory.

It would be interesting to reproduce the situation, when we get the port clash here. Maybe we observed it many times but didn't know the reason.

Reply:
Ok, it was trivial to switch to free port auto resolving in `TarantoolInspector`. Fixed.

Reply:
Nice! (It would be good to mark the new two commits as part of/related/fixes the issue.)